wavesrv/pkg/remote/openai/openai.go

```diff
@@ -72,6 +79,60 @@ func RunCompletion(ctx context.Context, opts *sstore.OpenAIOptsType, prompt []ss
 	return marshalResponse(apiResp), nil
 }
 
+func RunCloudCompletionStream(ctx context.Context, opts *sstore.OpenAIOptsType, prompt []sstore.OpenAIPromptMessageType) (chan *packet.OpenAIPacketType, error) {
+	const AWSLambdaCentralWSAddr = "wss://5lfzlg5crl.execute-api.us-west-2.amazonaws.com/dev/"
+	if opts == nil {
+		return nil, fmt.Errorf("no openai opts found")
+	}
+	if opts.Model == "" {
+		return nil, fmt.Errorf("no openai model specified")
+	}
+	websocketContext, _ := context.WithTimeout(context.Background(), CloudWebsocketConnectTimeout)
+	conn, _, err := websocket.DefaultDialer.DialContext(websocketContext, AWSLambdaCentralWSAddr, nil)
+	if err != nil {
+		log.Printf("Websocket error: %v", err)
+		return nil, fmt.Errorf("Websocket error: %v", err)
+	}
+	cloudCompletionRequestConfig := sstore.OpenAICloudCompletionRequest{
+		Prompt:     prompt,
+		MaxTokens:  opts.MaxTokens,
```

Comment:
I still let the user specify max tokens and max choices for the cloud, should I use the defaults instead or is this ok?

Reply:
I think we shouldn't let the user specify these.  We should just set them in the cloud.